.deepsource.toml

```diff
@@ -0,0 +1,17 @@
+version = 1
+
+test_patterns = [
+  'tests/**'
+]
+
+exclude_patterns = [
+
+]
+
+[[analyzers]]
+name = 'python'
+enabled = true
+runtime_version = '3.x.x'
+
+  [analyzers.meta]
```

Comment:
Tabulation issue?

Reply:
I think it is automatically generated this way: https://github.com/BloodAxe/pytorch-toolbelt/blob/develop/.deepsource.toml